PS4k/src/com/vitco/layout/content/menu/MainMenuLogic.java

```diff
@@ -730,6 +785,206 @@ protected Object doInBackground() throws Exception {
 
                         // ===========
 
+                    } else if (dialog.is("export_type=turntable_image_sequence")) {
+
+                        // ===========
+
+                        // -- export turntable image sequence
+                        // check if folder is empty
+                        // went with this approach cause it's "cheaper"
+                        try {
+                            final File[] fileList = toValidateFolder.getParentFile().listFiles();
+                            if (fileList != null && fileList.length > 0) {
+                                if (JOptionPane.showConfirmDialog(frame,
+                                        langSelector.getString("folder_not_empty_query"),
+                                        langSelector.getString("folder_not_empty_query_title"),
+                                        JOptionPane.OK_CANCEL_OPTION) != JOptionPane.OK_OPTION) {
+                                    return false;
+                                }
+                            }
+                        } catch (Exception e) {
+                            errorHandler.handle(e);
+                        }
+
+                        // create progress dialog
+                        final ProgressDialog progressDialog = new ProgressDialog(frame);
+                        progressDialog.start(new ProgressWorker() {
+                            @Override
+                            protected Object doInBackground() throws Exception {
+
+                                // export color render (image)
+                                progressDialog.setActivity("Processing turntable image sequence", true);
```

Comment:
Yes. There are other places in the code where I do that. You need to have a counter and also provide the max value.